UI/src/view/point_asset/trafficSignForm.js

```diff
@@ -231,22 +231,42 @@
       );
     }
 
+    var hideOldCodes = function (oldValues, propertyValue, groupPropertyValue) {
+      if (_.includes(oldValues, groupPropertyValue) && propertyValue == groupPropertyValue) {
+        //if true it will return false at the end.
+      }
+      else if (_.includes(oldValues, groupPropertyValue))
+        return true;
+
+      return false;
+    };
+
+    var sortByPropertyDisplay = function(collection) {
+      return collection.sort(function(element1, element2) {
+        return element1.propertyDisplayValue.localeCompare(element2.propertyDisplayValue, undefined, {numeric: true, sensitivity:'base'});
+      });
+    };
+
     var singleChoiceSubType = function (collection, mainType, property) {
       var propertyValue = (_.isUndefined(property) || property.values.length === 0) ? '' : _.head(property.values).propertyValue;
       var propertyDisplayValue = (_.isUndefined(property) || property.values.length === 0) ? '' : _.head(property.values).propertyDisplayValue;
       var signTypes = getValuesFromEnumeratedProperty ('trafficSigns_type');
       var groups = collection.getGroup(signTypes);
       var subTypesTrafficSigns;
+      var oldValues = ["143", "162", "166", "167", "168", "247", "274", "287", "288", "357", "359"];
```

Comment:
I think is better to move this values to the trafficSignColletion since it makes more sense there

Reply:
Done.

Reply:
Done.

Reply:
Done.